pennylane/ops/qubit/state_preparation.py

```diff
@@ -127,9 +129,15 @@ def state_vector(self, wire_order: Optional[WiresLike] = None) -> TensorLike:
 
 
 class StatePrep(StatePrepBase):
-    r"""StatePrep(state, wires)
+    r"""StatePrep(state, wires, pad_with, nomralize)
     Prepare subsystems using the given ket vector in the computational basis.
 
+    By setting ``pad_with`` to a real or complex number, ``state`` is automatically padded to dimension
```

Comment:
What does this mean? 🤔 What is the `embedding` you mention here?

Reply:
Changed to "template"